src/universal/mutations/UpdateUserProfileMutation.js

```diff
@@ -0,0 +1,50 @@
+import {commitMutation} from 'react-relay';
+
+graphql`
+  fragment UpdateUserProfileMutation_notification on UpdateUserProfilePayload {
+    user {
+      preferredName
+      picture
+    }
+  }
+`;
+
+graphql`
+  fragment UpdateUserProfileMutation_teamMember on UpdateUserProfilePayload {
+    teamMembers {
+      preferredName
+      picture
+    }
+  }
+`;
+
+const mutation = graphql`
+  mutation UpdateUserProfileMutation($updatedUser: UpdateUserProfileInput!) {
+    updateUserProfile(updatedUser: $updatedUser) {
+      ...UpdateUserProfileMutation_notification @relay(mask: false)
```

Comment:
Help me understand why:

- ... we're declaring fragments in this mutation module that are then getting imported in a subscription module
- ... we're masking these fragments on the mutation? Presumably we don't have fragment containers that are querying data directly off of a mutation response.

Reply:
1. it's DRY. mutations & subs want the same payload. why not use a fragment to ensure they get the same payload?
2. we have to mask every fragment on a mutation because Relay lies about their `onCompleted` callback receiving a raw result. if it's masked, then the onCompleted callback will fail.Since I'm not sure which mutations will use onCompleted, i just put it on all of them, because that's a super annoying error.

Reply:
1. DRY is good! I'm less concerned here about DRY and more about separation of concerns. Just pointing out that when we share code between two modules, if that shared code isn't biased towards one of it's use cases over the other, then it should probably be in a neutral third location.